www/main/settings/components/ThemeContainer.jsx

```diff
@@ -81,8 +81,12 @@ const ThemeContainer = () => {
                 onRemove={() => {
                   removeCustomBackgroundFile(tile['background-image-path'].replace(/\\(\s)/g, ' '));
                   upsertCustomBackgrounds(setCustomThemes);
-                  const currentThemeInstance = themes.filter((theme) => theme.key === currentTheme);
-                  setDefaultBg(currentThemeInstance, setThemeBg, themeBg);
+                  if (tile['background-image'] === themeBg.url.href) {
+                    const currentThemeInstance = themes.filter(
+                      (theme) => theme.key === currentTheme,
+                    );
+                    setDefaultBg(currentThemeInstance, setThemeBg, themeBg);
```

Comment:
The scope of `currentThemeInstance` variable is inside the if statement only and it is being used outside of that if statement which is a violation. Veerji this should be fixed IMO

Reply:
It is inside if statement, if statement, ends at line 89.

Reply:
I messed up veerji @Gauravjeetsingh really sorry for this 🙏